### ISSUE TITLE
[using-drupal] demo doesn't work with Drupal 8.7

### ISSUE THREAD

**Issue by JohnAlbin** [gatsbyjs/gatsby#15693]
## Description

The Using Drupal demo is hard-coded to use the JSON:API endpoints at `https://live-contentacms.pantheonsite.io/api`. This site is using Drupal 8.6.x or earlier and not the latest Drupal 8.7.0 series which has the new, stable JSON:API module.

When using the using-drupal files and editing the Gatsby config to use a Drupal 8.7.x site, the files report multiple errors because the endpoint structure wasn't stable with Drupal 8.6.x and has changed now that 8.7.0 has been released.

### Steps to reproduce

1. Use the `examples/using-drupal` files and run `npm install` to install Gatsby.
2. Update the `gatsby-config.js` file and change the `gatsby-source-drupal` plugin configuration to use a `baseUrl` and `apiBase` that point to a Drupal 8.7.x site. (Or, if you own the `https://live-contentacms.pantheonsite.io/` website, update Drupal to the latest version.)
3. Run `npm run build` to build the static Gatsby site.

### Expected result

The Gatsby site is built successfully.

### Actual result

The `gatsby build` process errors out with multiple graphQL errors occur.

### Environment

```
  System:
    OS: macOS 10.14.5
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
    Yarn: 1.16.0 - ~/.yarn/bin/yarn
    npm: 6.9.2 - ~/.nvm/versions/node/v10.16.0/bin/npm
  Languages:
    Python: 2.7.10 - /usr/bin/python
  Browsers:
    Chrome: 75.0.3770.100
    Firefox: 68.0
    Safari: 12.1.1
  npmPackages:
    gatsby: ^2.0.0 => 2.13.20
    gatsby-image: ^2.0.5 => 2.2.6
    gatsby-plugin-glamor: ^2.0.5 => 2.1.3
    gatsby-plugin-google-analytics: ^2.0.5 => 2.1.4
    gatsby-plugin-offline: ^2.0.5 => 2.2.4
    gatsby-plugin-sharp: ^2.0.5 => 2.2.7
    gatsby-plugin-typography: ^2.2.0 => 2.3.2
    gatsby-source-drupal: ^3.0.9 => 3.2.7
    gatsby-transformer-sharp: ^2.1.1 => 2.2.3
  npmGlobalPackages:
    gatsby-cli: 2.7.7
```

**Comment by eojthebrave** [comment]
Given that JSON:API is in Drupal core now, and you can therefore use Gatsby with Drupal without any additional Drupal conrib modules. I wonder if it makes sense to switch this demo code to use a Drupal site that is "just" core + the Umami demo profile, and not one based on Contenta. Which is core + a whole bunch of additional things that are useful if you're all in on using Drupal as the a headless CMS. But not at all required if what you're trying to do is test out the Drupal + Gatsby integration.

**Comment by JohnAlbin** [comment]
I agree with @eojthebrave here. I talked with Mateu (@e0ipso) yesterday about the Contenta website and its JSON:API endpoint and he said that if a developer wanted to use the `using-drupal` code with a vanilla Drupal 8.7 website (with Umami content), that it may or may not work 100%.

Correct me if I misunderstood something, Mateu.

So we'll need to provide a new endpoint with a vanilla Drupal 8.7 (Umami profile) install. I've started talking to amazee.io about donating a public website for this purpose. I'll report back end of next week as I'm traveling internationally on Monday.
